src/main/java/com/arxanfintech/sdk/wallet/Wallet.java

```diff
@@ -391,4 +391,39 @@ private String ProcessTx(JSONObject jsonheader, JSONObject jsonbody) {
             return e.getMessage();
         }
     }
+
+    public String IndexSet(JSONObject jsonheader, JSONObject jsonbody) {
+        Request request = new Request();
+        request.client = this.client;
+        request.body = jsonbody;
+        request.header = jsonheader;
+        request.crypto = crypto;
+        request.url = "http://" + request.client.Address + "/wallet-ng/v1/index/set";
```

Comment:
You cannot assume that the wallet service must be accessed through the wasabi gateway, so should not hard code the 'wallet-ng' route tag at here. pls refer the wallet-sdk-go to set route tag.

Reply:
done